charts/k8s-monitoring/docs/examples/destinations/otlp-gateway/values.yaml

```diff
@@ -0,0 +1,28 @@
+---
+cluster:
+  name: otlp-gateway-test
+
+destinations:
+  - name: otlp-gateway
+    type: otlp
+    url: https://otlp-gateway-my-region.grafana.net/otlp
+    protocol: http
+    auth:
+      type: basic
+      username: my-gateway-username
+      password: my-gateway-password
+    metrics: {enabled: true}
+    logs: {enabled: true}
+    traces: {enabled: true}
```

Comment:
Just a question for my own understanding, is the `{enabled: true}` syntax specific to OTLP?